src/buildercore/trop.py

```diff
@@ -360,6 +360,16 @@ def render_s3(context, template):
             'DeletionPolicy': context['s3'][bucket_name]['deletion-policy'].capitalize()
         }
         bucket_title = _sanitize_title(bucket_name) + "Bucket"
+        if context['s3'][bucket_name]['cors']:
+            props['CorsConfiguration'] = s3.CorsConfiguration(
+                CorsRules=[
+                    s3.CorsRules(
+                        AllowedHeaders=['*'],
+                        AllowedMethods=['GET', 'HEAD'],
+                        AllowedOrigins=['*']
```

Comment:
that's called `elife-bot` and you rightly often elaborated on its level of internal quality :)